common/domain/dlqMessageHandler_test.go

```diff
@@ -22,14 +22,19 @@ package domain
 
 import (
 	"context"
+	"errors"
```

Comment:
let's change the file names (test file and corresponding code file) to snake_case as we go.